test/websocket.test.js

```diff
@@ -1774,6 +1774,21 @@ describe('WebSocket', () => {
       assert.strictEqual(ws.listenerCount('foo'), 0);
     });
 
+    it('has a stub `dispatchEvent` method', (done) => {
+      const wss = new WebSocket.Server({ port: 0 }, () => {
```

Comment:
We can simplify this test, there is no need to establish a connection:

```js
const ws = new WebSocket('ws://localhost', {
  lookup() {}
});

assert.throws(
  ws.dispatchEvent(),
  /^Error: The dispatchEvent\(\) method is not implemented$/
)
```

Reply:
The `WebSocket` in this scope is a WebSocket from the jsdom lib. I think this test would fail?

Reply:
Never mind, this works. What I found earlier was that passing a fake event would cause another test to fail for some reason.